### PR TITLE
FOUR-17044 an add button appears when doing d&d

### PR DESCRIPTION
**Pull request by devmiguelangel**
## Issue & Reproduction Steps

Expected behavior: 
An add button should not appear when doing drag and drop in Calcs and watchers

Actual behavior: 
An add button appears when doing drag and drop in Calcs and watchers

## Solution
- Hide the plus icon
- Add tooltips to edit and delete buttons

## How to Test
1. Import attached secreen
2. Go to designer tab
3. Search screen and edit
4. First scenario: Calcs
5. Click on Calcs button
6. Drag and drop last Calc(C3) to top

## Related Tickets & Packages
[FOUR-17044](https://processmaker.atlassian.net/browse/FOUR-17044)

## Code Review Checklist
- [ ] I have pulled this code locally and tested it on my instance, along with any associated packages.
- [ ] This code adheres to [ProcessMaker Coding Guidelines](https://github.com/ProcessMaker/processmaker/wiki/Coding-Guidelines).
- [ ] This code includes a unit test or an E2E test that tests its functionality, or is covered by an existing test.
- [ ] This solution fixes the bug reported in the original ticket.
- [ ] This solution does not alter the expected output of a component in a way that would break existing Processes.
- [ ] This solution does not implement any breaking changes that would invalidate documentation or cause existing Processes to fail.
- [ ] This solution has been tested with enterprise packages that rely on its functionality and does not introduce bugs in those packages.
- [ ] This code does not duplicate functionality that already exists in the framework or in ProcessMaker.
- [ ] This ticket conforms to the PRD associated with this part of ProcessMaker.

ci:next

[FOUR-17044]: https://processmaker.atlassian.net/browse/FOUR-17044?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by rodriquelca — APPROVED** [review]

File: src/components/sortable/sortableList/SortableList.vue [hunk 1]
```diff
@@ -65,6 +65,7 @@
             </button>
             <button
               v-else
+              v-b-tooltip="{ customClass: 'sortable-item-action-btn-tooltip' }"
               class="btn"
               title="Edit"
```
Comment (caleeli) [inline src/components/sortable/sortableList/SortableList.vue:70]: translation is missing here

File: src/components/sortable/sortableList/SortableList.vue [hunk 2]
```diff
@@ -75,6 +76,7 @@
             <div class="sortable-item-vr"></div>
             <slot name="options" :item="item"></slot>
             <button
+              v-b-tooltip="{ customClass: 'sortable-item-action-btn-tooltip' }"
               class="btn"
               title="Delete"
```
Comment (caleeli) [inline src/components/sortable/sortableList/SortableList.vue:81]: translation for this label

File: src/components/sortable/sortableList/SortableList.vue (hunk 1 above)
Comment (devmiguelangel) [reply src/components/sortable/sortableList/SortableList.vue]: It was fixed

File: src/components/sortable/sortableList/SortableList.vue (hunk 2 above)
Comment (devmiguelangel) [reply src/components/sortable/sortableList/SortableList.vue]: It was fixed

**Review by caleeli — APPROVED** [review]

**Pull request merged**